app/components/file-menu.js

```diff
@@ -33,13 +33,15 @@ export default Ember.Component.extend({
       prompt('Ctrl + C ;-)', window.location.href);
     },
     embed() {
-      let src = window.location.href.split("?")[0];
-      src += "?fullScreen=true";
-      let iframe = document.createElement("iframe");
+      let src = window.location.href.split('?')[0];
+      src += '?fullScreen=true';
+      let responsive = document.createElement('div');
+      responsive.style.cssText = 'position:relative;height:0;overflow:hidden;max-width:100%;padding-bottom:56.25%;'; // 16:9
```

Comment:
Many people who are embedding this in a page will want to specify the width and the height. How would they do this?